ui/src/pages/Create.js

```diff
@@ -123,6 +132,12 @@ const MESSAGES = defineMessages({
     description: ('A label for a form field where users can submit a photo URL '
         + '(as opposed to uploading a photo directly)'),
   },
+  otherPersonMessageExplanationText: {
+    id: 'Create.otherPersonMessageExplanationText',
+    defaultMessage: ('Write a message for this person, or for others seeking '
+        + 'this person.'),
+    description: 'Hint text for a form field for a message for/about a person.',
```

Comment:
This description could describe the purpose of the form field a little more clearly?

Reply:
Done.